### PR TITLE
[Publisher] Home + HomeStore: Tests

### PR DESCRIPTION
**Pull request by mxosman**
## Description of the change

Adds basic unit/integration tests for the Home component and HomeStore.

## Related issues

Contributes to #849

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [x] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by mxosman** [comment]
This is just a starting point - we can always add more mocks and cases to test, but this should give some reasonable coverage for the Home & HomeStore.

File: publisher/src/stores/FormStore.ts [hunk 1]
```diff
@@ -414,7 +414,7 @@ class FormStore {
     reportID: number,
     metricKey: string,
     updatedValue: string,
-    metricEnabled: boolean | undefined
+    metricEnabled: boolean | undefined | null
```
Comment (alfonsogoberjr) [inline publisher/src/stores/FormStore.ts:417]: nit: can we get away with `metricEnabled?: boolean` somehow with `false` as a default?

File: publisher/src/components/Home/Home.test.tsx [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+  updateRecordProps,
+} from "../../mocks/HomeMocksHelpers";
+import { rootStore, StoreProvider } from "../../stores";
+import { Home } from "./Home";
+
+const { homeStore, userStore, authStore } = rootStore;
+const mockAgencyID = "10";
+
+beforeEach(() => {
+  runInAction(() => {
```
Comment (alfonsogoberjr) [inline publisher/src/components/Home/Home.test.tsx:37]: Nice

**Review by alfonsogoberjr — APPROVED** [review]

File: publisher/src/stores/FormStore.ts (hunk 1 above)
Comment (mxosman) [reply publisher/src/stores/FormStore.ts:417]: Good question! So `metricEnabled` being `true`, `false`, and `null` have 3 different meanings. True/False is whether or not its enabled, and `null` is how we know a metric has never been set/touched/configured. [We should've updated the types a while back - not sure how we missed that!]

**Review by lilidworkin — APPROVED** [review]

**Pull request merged**